sysconfig/jureca-booster/modules.yaml

```diff
@@ -27,6 +27,7 @@ modules:
           '+profile': 'profile'
           '^coreneuron+knl': 'knl'
           'neuron~mpi': 'serial'
+          '^python@3.6.8': 'python3'
```

Comment:
Why?

Reply:
We need it for
https://github.com/cnr-ibf-pa/hbp-bsp-issues/issues/525#issuecomment-590703254